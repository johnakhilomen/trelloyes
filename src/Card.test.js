import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Card from './card';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Card />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('renders markup as expected without props', () => {
    // Render the component, as JSON
    const tree = renderer.create(<Card />).toJSON();
    // Check whether it matches the previous snapshot
    // Stored in __snapshots__/Card.test.js.snap
    expect(tree).toMatchSnapshot(); 
});