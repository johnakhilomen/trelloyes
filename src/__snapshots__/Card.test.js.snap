// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders markup as expected without props 1`] = `
<div
  className="Card"
>
  <button
    type="button"
  >
    delete
  </button>
  <h3 />
  <p />
</div>
`;
